package.json

```diff
@@ -22,10 +22,12 @@
     "angular-toarrayfilter": "1.0.3",
     "angular-ui-router": "0.4.3",
     "angular1-ui-bootstrap4": "2.4.22",
+    "auth0-js": "^9.0.2",
     "bootstrap": "4.0.0-beta.2",
     "deepmerge": "1.5.2",
     "eslint-plugin-react": "7.5.1",
     "font-awesome": "4.7.0",
+    "got": "8.0.3",
```

Comment:
What are you thoughts on:
https://npmcompare.com/compare/axios,fetch,got,request,superagent

...particularly for consumers that only need to support recent browsers (eg our babel config only specifies "last 1 version" of the main 4).

That said, it's only used in the function that might be removed, so we might not need to use any library after all :-)

Reply:
@edmorley In that case, I could just rely on https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API. No need to bring in a new library :)